recipes/default.rb

```diff
@@ -17,13 +17,8 @@
                     kubernetes_domain_name = node['kube-hops']['cluster_domain']
                 end
             end
-
-        end
-        if node['install']['localhost'].casecmp?("true")
-            my_ip = "127.0.0.1"
```

Comment:
This variable is used in another places in the recipe

Reply:
My bad